lib/puppetserver/utils/signing_digest.rb

```diff
@@ -0,0 +1,25 @@
+module Puppetserver
+  module Utils
+    class SigningDigest
+
+      attr_reader :errors, :digest
+
+      def initialize
```

Comment:
I think since this is a specific utility object, it sorta makes sense. An alternative might be a module method grouped with other SSL utils (if there are any), just called `digest` perhaps, that returns a (possibly memoized) digest object. Depends on how important it is to us to reuse the object I guess. But I don't mind this.